joomla/joomla-cms
# Fixes php 8.1 strtoupper deprecation error in patch tester

Pull Request for Issue # .

### Summary of Changes
Removes the warning 
``` Deprecated: strtoupper(): Passing null to parameter #1 ($string) of type string is deprecated in E:\Bearsampp\www\j4\libraries\src\MVC\Model\ListModel.php on line 543 ``` that shows up in patch_tester 4.2.1 when using php 8.1+


### Testing Instructions

install https://github.com/joomla-extensions/patchtester/releases/tag/4.2.1
when you open it you will see the error at the top of the page.
enter your credentials and apply this patch.


### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/194698762-5de55f01-34f9-4b72-b80e-585129c90006.png)



### Expected result AFTER applying this Pull Request

![image](https://user-images.githubusercontent.com/1850089/194698804-e870a90d-a9b7-421a-b52f-1ce7ce9784af.png)


### Link to documentations
Please select:
